docs/deployment/linux.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Deployment on a Linux Server
+---
+
+Deploying Velocity on a Linux machine is easy! This guide will walk you through the installation of Velocity.
```

Comment:
nit: avoid using terms like "easy" or "just" in guides. Here's a more [comprehensive list](https://css-tricks.com/words-avoid-educational-writing/).